backend/src/main/resources/application.properties

```diff
@@ -2,4 +2,13 @@ spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto = update
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
 
-spring.profiles.active=dev
\ No newline at end of file
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=invsofttest@gmail.com
```

Comment:
Please, move credentials to `application-dev.properties` profile, cause they will change between environments